vsts/vsts.yaml

```diff
@@ -313,7 +318,7 @@ jobs:
           E2E_IKEY: $(E2E-IKEY)
 
           # Environment variables for the DevOps pipeline
-          PROXY_SERVER_ADDRESS: 127.0.0.1:8888
+          PROXY_SERVER_ADDRESS: 127.0.0.1:3128
```

Comment:
port needs to be updated [here](https://github.com/Azure/azure-iot-sdk-csharp/blob/ac3e71036edd0a099c5c7283628085a252d4c01a/vsts/vsts.yaml#L284)?